front-end/src/components/modal/create/GraphSettings.js

```diff
@@ -27,20 +27,21 @@ const GraphSettings = ({ movePage, setGraphType, setLiveCheck }) => {
         <div className="graphTypesBox">
           <div className="graphOptionsText">Graph Types</div>
           <div className="GraphTypeSelect">
-            <select id="graphTypeSelect" >
+            <select id="graphTypeSelect" value={graphType} onChange={getGraphType}>
```

Comment:
These graphType changes are not necessary and confusing. Should only need setGraphType, and can use your own getGtaphType function to fetch it in this file, not passed as a prop. Also not needed in this line or elsewhere. Disable the live button makes sense, but you can just set its value back to false rather than passing liveCheck down too

Reply:
I don't understand. It is required in my opinion to handle changing the state of the live check?